lib/workers/branch/commit.ts

```diff
@@ -31,9 +31,20 @@ export function commitFilesToBranch(
   }
   const fileLength = [...new Set(updatedFiles.map((file) => file.name))].length;
   logger.debug(`${fileLength} file(s) to commit`);
-  // istanbul ignore if
   if (getAdminConfig().dryRun) {
-    logger.info('DRY-RUN: Would commit files to branch ' + config.branchName);
+    logger.info(
+      {
+        commitMessage: config.commitMessage,
+
+        // Piping f.contents to
+        // git diff -- ${f.name} -
+        // would be much better, but this seems to be impossible with simple-git
+        updatedFiles: `\n${updatedFiles
+          .map((f) => `${f.name}\n<<<\n${String(f.contents)}\n>>>\n`)
```

Comment:
This would be too verbose in many cases. e.g. lock files can be huge

Reply:
Could we add a flag or something? I don't see any easy way to extract all necessary information across all managers.

Reply:
I think logger.info could print the equivalent of `git status` and logger.debug could print the equivalent of `git diff`. It's better not to introduce any new flag and have a *complex* dry-run.

Reply:
`logger.debug` also outputs tons of other information that are not relevant if you are not debugging.
The names of the changed files on the other hand don't provide the information which updates would be performed, so it is not really helpful.

The information I was trying to get out of the dry-run is how running renovate with a given config would alter the dependencies in the repo. Currently it is hidden in a lot of noise about what renovate is doing internally.
I'd like to get a log of all the effects, but only the effects.